cmd/mdatagen/loader.go

```diff
@@ -124,11 +124,7 @@ func (m *metric) Unmarshal(parser *confmap.Conf) error {
 	if !parser.IsSet("enabled") {
 		return errors.New("missing required field: `enabled`")
 	}
-	err := parser.Unmarshal(m)
```

Comment:
I think this one I already merged on a different PR, right? Can you rebase your branch?

Reply:
Done, thanks